lib/iex/lib/iex/autocomplete.ex

```diff
@@ -265,24 +265,37 @@ defmodule IEx.Autocomplete do
   end
 
   defp get_module_funs(mod) do
-    if function_exported?(mod, :__info__, 1) do
-      if docs = Code.get_docs(mod, :docs) do
-        Enum.filter_map(docs, &has_content?/1, &elem(&1, 0))
-      else
-        mod.__info__(:macros) ++ (mod.__info__(:functions) -- [__info__: 1])
-      end
+    docs = Code.get_docs(mod, :docs) || []
+    Enum.map(docs, &elem(&1, 0)) ++ module_info_funs(mod)
```

Comment:
@josevalim I just updated the PR. It was indeed much cleaner to make `Kernel.SpecialForms` an exception, thanks for the advice!